brave/brave-core
# several shields panel improvements

## Submitter Checklist:

fix https://github.com/brave/brave-browser/issues/1894
fix https://github.com/brave/brave-browser/issues/1892
fix https://github.com/brave/brave-browser/issues/2488
fix https://github.com/brave/brave-browser/issues/2565

## Test Plan:

1. Ensure all features on shields work
  - Ads/Trackers should bump in sites such as https://wired.com
  - HTTPSE should respond to https://https-everywhere.badssl.com
  - Cookies should respond to http://www.html-kit.com/tools/cookietester/
  - Fingerprinting should respond to https://codepen.io/run-time/pen/XJNXWV

2. Ensure shields is accessible via keyboard using <kbd>tab</kbd> for navigation, <kbd>space</kbd> for actions. <kbd>enter</kbd> should work for buttons as well.

3. Ensure the general (not scripts) blocked list (accessed via arrow down icon) can be scrolled both horizontally and vertically

4. Issue brave/brave-browser#2488 is being set as fixed as scripts now use a toggle instead of a select element

5. Shields should look like the original spec [available here](https://brave.github.io/brave-ui/?selectedKind=Feature%20Components%2FShields&selectedStory=Enabled&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel&background=%23E9ECEF)

6. Ensure that once a given resource stat reach 0, the option to show resources blocked should have no interactions other than toggling switch on/off

## Reviewer Checklist:
